<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(163, 193, 173);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(16, 16, 16);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(163, 193, 173);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 12px;
padding-left: 5px;
padding-right: 5px;
</string>
    </property>
    <property name="text">
     <string>Get ROI</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 7;
padding: 3px;
font-size: 13px;
padding-left: 5px;
padding-right: 5px;
qproperty-alignment: AlignCenter,AlignVCenter ;
</string>
    </property>
    <property name="text">
     <string>Number of Screen</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="numberOfScreen">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>69</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 1px;
background-color:rgb(163, 193, 173);
border-color:rgb(163, 193, 173);
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 12px;
padding-left: 5px;
padding-right: 5px;
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="screemComboBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>105</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(16, 16, 16);
background-color: rgb(231, 231, 231);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 10;
padding: 3px;
font-size: 12px;
padding-left: 5px;
padding-right: 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="numberOfScreen_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 7;
padding: 3px;
font-size: 13px;
padding-left: 5px;
padding-right: 5px;</string>
    </property>
    <property name="text">
     <string>Screen Selection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="captureVideo">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(16, 16, 16);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(163, 193, 173);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 12px;
padding-left: 5px;
padding-right: 5px;
</string>
    </property>
    <property name="text">
     <string>As Screen Capture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="screenShot">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(16, 16, 16);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(163, 193, 173);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 12px;
padding-left: 5px;
padding-right: 5px;
</string>
    </property>
    <property name="text">
     <string>Take As Screenshot</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 7;
padding: 3px;
font-size: 22px;
padding-left: 5px;
padding-right: 5px;
qproperty-alignment: AlignCenter,AlignVCenter ;
</string>
    </property>
    <property name="text">
     <string>SRecorder</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
